common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -103,7 +106,12 @@ case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
           child.attr("src", facebookUrl)
         }
 
-        child.wrap("<div class=\"embed-video-wrapper u-responsive-ratio u-responsive-ratio--hd\"></div>")
+        val someIframe = Option(element.select("iframe").first())
+
+        someIframe match {
+          case Some(iframe) => wrapIframe(child, iframe)
+          case None         => wrapHD(child)
+        }
```

Comment:
Matching Optional can usually be written more concisely using `.map().getOrElse()`. 
```
Option(element.select("iframe").first())
   .map(iframe => wrapIframe(child, iframe))
   .getOrElse(wrapHD(child))
```

Reply:
This is my first Scala so I tried to keep it idiomatic! I read that pattern matching was a touch more readable but having since done some Googling it seems that `map getOrElse` (less so `fold`) seems to be the preferred way so thanks!

Reply:
That's an open debate 😬 . When the logic is more complex pattern matching might be easier to read. Here the logic is very simple.